torch/_inductor/codegen/triton.py

```diff
@@ -1820,9 +1820,25 @@ def codegen_kernel(self, name=None):
 
         code = IndentedBuffer()
 
-        size_hints = [
-            next_power_of_2(V.graph.sizevars.size_hint(numel)) for numel in self.numels
-        ]
+        size_hints = []
+        for numel in self.numels:
+            numel_hint = V.graph.sizevars.symbolic_hint(numel)
+            if not isinstance(numel_hint, (int, sympy.Integer)):
```

Comment:
is `not (int, sympy.Integer)` the same things as 'unbacked symint'? so in other words, this is the thing we do when nobody gave a size hint?